src/java.base/share/classes/java/nio/file/InvalidPathException.java

```diff
@@ -39,7 +39,15 @@
     @java.io.Serial
     static final long serialVersionUID = 4355821422286746137L;
 
+    /**
+     * The invalid input path string.
+     */
```

Comment:
This should probably be "The input string" to be consistent with the getInput method.